contracts/state/PoseidonSMT.sol

```diff
@@ -124,7 +127,11 @@ contract PoseidonSMT is Initializable, TSSUpgradeable {
     }
 
     function _notifyRoot() internal {
```

Comment:
Don't really like the naming of this function. We are no longer "just notifying".

Reply:
_commitRoot() ?

Reply:
Updated to _commitRoot